test/TestUtils/project.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "compile": {
+      "include": [
+        "../../build_projects/shared-build-targets-utils/Utils/DotNetCli.cs",
+        "../../build_projects/shared-build-targets-utils/Utils/EnvVars.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/AnsiConsole.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/AnsiColorExtensions.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/ArgumentEscaper.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/BuildFailureException.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/BuildReporter.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/Command.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/CommandResult.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/Constants.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/Reporter.cs"
+      ]
+    }
+  },
+
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0-rc3-004338"
```

Comment:
Thinking through this a little more, the sdk will run on it's version of Microsoft.NETCore.App. The host is RTMing and must be backward compatible, so if it makes a breaking change a test should fail. 

The breaking-ness is really limited to dotnet-test-xunit as that is the only piece which will run under the latest Microsoft.NETCore.App, and if the host breaks that scenario, tests should fail. So I've come to agree, these should be on the latest Microsoft.NETCore.App. We need to avoid running an SDK on this latest version, as that causes the circular breakage, but that's not what is happening when we make these tests refererence `-*`

---
In reply to: [66108594](https://github.com/dotnet/core-setup/pull/97#discussion_r66108594) [](ancestors = 66108594,66107545)